File: C++/057. DivisaoExata/DivisaoExata.cpp
#include <iostream>
int main()
{
    long long A, B;
    std::cin >> A >> B;
    if (A % B == 0) {
        std::cout << "a divisao de " << A << " por " << B  << " eh exata." << std::endl;
        return 0;
    }
    return 0;
}

File: Java/057. DivisaoExata/DivisaoExata.java
import java.util.Scanner;
public class DivisaoExata
{
    public static void main(String[] args) 
    {
        Scanner scan = new Scanner(System.in);
        long X = scan.nextLong();
        long Y = scan.nextLong();
        if (X % Y == 0)
        {
            System.out.println("a divisao de " + X + " por " + Y + " eh exata.");
        }
        scan.close();   
    }
}